ghostwriter: 2.1.5 -> 2.2.0

erictapen opened the pull request:
###### Description of changes

[Changelog](https://github.com/wereturtle/ghostwriter/releases/tag/2.1.6)
[Changelog](https://github.com/wereturtle/ghostwriter/releases/tag/2.2.0)

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

erictapen committed:
ghostwriter: 2.1.5 -> 2.2.0

DeeUnderscore commented on pkgs/applications/editors/ghostwriter/default.nix:
```diff
@@ -14,13 +14,13 @@
+  version = "2.2.0";
 
   src = fetchFromGitHub {
     owner = "wereturtle";
```
Ghostwriter is apparently becoming a KDE project, and the new project URL is <https://github.com/KDE/ghostwriter>, so we can change this to `"KDE"`, and also alter `meta.changelog`.

(Looks like it will eventually [move to the KDE Gitlab](https://github.com/KDE/ghostwriter/issues/821#issuecomment-1257335313), but that's not done yet)

erictapen replied:
Good catch, I added that.

DeeUnderscore reviewed (changes requested):
Besides the URL changes, looks good. 

Result of `nixpkgs-review pr 191168` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ghostwriter</li>
  </ul>
</details>

erictapen committed:
ghostwriter: change github owner and homepage, as it is now a KDE project

The Github Repo already redirects and the new homepage is mentioned
here: https://github.com/KDE/ghostwriter/issues/821#issuecomment-1258100792
